fgpyo/fasta/reference_set_builder.py

```diff
@@ -0,0 +1,174 @@
+"""
+Classes for generating Fasta files and records for testing
+----------------------------------------------------------------
+"""
+# import hashlib
+import os
+import textwrap
+from tempfile import NamedTemporaryFile
+from typing import List
+from typing import Optional
+
+# pylint: disable=W0511
+
+
+class ReferenceSetBuilder:
+    """
+    Builder for constructing one or more fasta records.
+    """
+
+    # The default asssembly
+    DEFAULT_ASSEMBLY: str = "testassembly"
+
+    # The default species
+    DEFAULT_SPECIES: str = "testspecies"
```

Comment:
Annotate as a class variable please (i.e. value used for all instances of the class, versus a value used for a specific instance of the class)
```suggestion
    DEFAULT_ASSEMBLY: ClassVar[str] = "testassembly"

    # The default species
    DEFAULT_SPECIES: ClassVar[str] = "testspecies"
```